bin/bazel_to_go.py

```diff
@@ -185,7 +185,7 @@ def bazel_to_vendor(WKSPC):
     with open(os.path.join(WKSPC, "lintconfig_base.json")) as fin:
         conf = json.load(fin)
     conf['exclude'].extend(manifest)
-    with open(os.path.join(WKSPC, "lintconfig.json"), "wt") as fout:
+    with open(os.path.join("/tmp", "istio-lintconfig.json"), "wt") as fout:
```

Comment:
this might not work on mac. I usually have seen on Mac the temp did is named `/T` and not `/tmp`

Reply:
If there is no easy way to put the file in tmp dir, we can just leave it to what you had before... adding the file to the .ignore file. May be renamed it to `istio-lintconfig.gen.json`

Reply:
Well then, that's a bit hard; `tempfile.gettempdir()` can be used here, but the same path needs to be specified in linters.sh script, I'm not sure how it can be specified in a platform agnostic way; let me revert back for that part, and use `lintconfig.gen.json` name (I don't think `istio-` prefix is needed in that case, obviously it's for istio when it's within the workspace).